package.json

```diff
@@ -11,11 +11,13 @@
   ],
   "dependencies": {},
   "devDependencies": {
```

Comment:
Yeah, I'm not sure what's the best approach here either. We should definitely include it as a devDep, at least, to be included in the example, and then state in the README that we depend on it?